.github/workflows/release.yml

```diff
@@ -8,7 +8,10 @@ on:
 jobs:
   release:
     name: release
-    runs-on: macOS-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
would it work to run only on `ubuntu-latest` and have a single `.goreleaser.yml` that has 2 builds, something like:

```
builds:
- id: saml2aws-linux
  main: ./cmd/saml2aws/main.go
  binary: saml2aws
  flags:
    - -tags=hidraw
    - -trimpath
    - -v
  ldflags:
    - -s -w -X main.Version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}}
  goos:
    - linux
  goarch:
    - amd64
    - arm64
    - arm
- id: saml2aws-non-linux
  main: ./cmd/saml2aws/main.go
  binary: saml2aws
  flags:
    - -trimpath
    - -v
  ldflags:
    - -s -w -X main.Version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}}
  goos:
    - windows
    - darwin
  goarch:
    - amd64
    - arm64
    - arm
```

You would also avoid having to conditionally install libudev-dev and also avoid having to use a matrix.

Reply:
hmmmm scratch that, you can't easily cross-compile with CGO enabled :disappointed: